README.md

```diff
@@ -31,22 +42,21 @@ Environment variables have **higher precedence** over initialization parameters.
 
 Check out the [configuration part](https://docs.thundra.io/python/configuration-options) of our docs for more detailed information.
 
-#### 1. Environment variables
+#### 1. Most Useful Environment variables
 
 | Name                                          | Type   |          Default Value           |
 |:----------------------------------------------|:------:|:--------------------------------:|
-| thundra_apiKey                                | string |                -                 |
-| thundra_agent_lambda_disable                  |  bool  |              false               |
-| thundra_agent_lambda_application_stage        | string |                -                 |
-| thundra_agent_lambda_trace_disable            |  bool  |              false               |
-| thundra_agent_lambda_metric_disable           |  bool  |              false               |
-| thundra_agent_lambda_log_disable              |  bool  |              false               |
-| thundra_agent_trace_request_skip              |  bool  |              false               |
-| thundra_agent_trace_response_skip             |  bool  |              false               |
-| thundra_agent_lambda_timeout_margin           |  int   |               200                |
-| thundra_agent_lambda_report_rest_baseUrl      | string |     https://api.thundra.io/v1    |
-| thundra_agent_lambda_report_cloudwatch_enable |  bool  |              false               |
-
+| THUNDRA_APIKEY                                | string |                -                 |
+| THUNDRA_APPLICATION_NAME                      | string |                -                 |
```

Comment:
except `THUNDRA_APIKEY`, all other env var should have must start with `THUNDRA_AGENT_...`